ui/apps/platform/src/services/ClustersService.ts

```diff
@@ -147,17 +133,6 @@ export function upgradeClusters(ids = []): Promise<Empty[]> {
     return Promise.all(ids.map((id) => upgradeCluster(id)));
 }
 
-/**
- * Fetches cluster by its ID.
- */
-// TODO specify return type after we rewrite without normalize
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-export function fetchCluster(id: string) {
-    return axios.get(`${clustersUrl}/${id}`).then((response) => ({
-        response: normalize(response.data, { cluster: clusterSchema }),
-    }));
-}
-
```

Comment:
do we just not use this and instead use `fetchClusterWithRetentionInformation` now?

Reply:
Yes, indeed. The other service function has additional information.
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Clusters/ClustersSidePanel.tsx#L131